proto/kvrpcpb.proto

```diff
@@ -151,13 +152,32 @@ message PrewriteRequest {
     uint64 start_version = 4;
     uint64 lock_ttl = 5;
     bool skip_constraint_check = 6;
+    // For pessimistic transaction, some mutations don't need to be locked, for example, non-unique index key.
+    repeated bool is_pessimistic_lock = 7;
```

Comment:
Embeded in Mutation is better?

Reply:
That would increase the unmarshal overhead for the non-pessimistic transactions.

Reply:
Why?

Reply:
the Mutation struct is bigger.